applications/aihorde-application.md

```diff
@@ -0,0 +1,202 @@
+monopoly of this powerful technology by allowing the more optimal sharing
+of resources and therefore the cooperation of human talent.
+
+I did this project as Free/Libre and Open Source so that it can survive any
+challenges, even from myself.  
+
+I also want to create a community of AI Enthusiasts which is explicitly
+**ethical and positive**. The Kudos system I developed is aimed to not only
+reward people with access to GPUs, but also just good behavior all around.
+This is why people consistently gift kudos (i.e. priority) to others for things
+like sharing their artwork, or answering questions, or doing development.
+
+One big positive result to come out of the AI Horde, is the improving of the
+commons through [rewarding the rating of images](https://laion.ai/blog/laion-stable-horde/),
+for the training of new open GenAI models.
+
+### Intended Use
+
+The AI Horde is built to provide the central location where those who have the
+resources can share them with those who do not, in a secure and safe manner. As
+the AI horde is handling everything through REST API,it is meant to be compatible
+with everything and easy to build around.
+
+The intended use is that 
+
+1. People with spare GPU resources will share them
+1. People with a development background will create new integrations based on
+   the AI Horde. (Frontends, Bots, Games etc)
+1. People with no GPU, or with bigger demands than their own GPU can handle
+   (e.g. text2video) will use the shared resources on the horde to utilize
+   generative AI for any purpose.
+
+### Anticipated Misuse
+
+1. For-profit or scammy services integrating into the horde and charging
+   their users for something available for free, effectively exploiting
+   the shared resources of the community for their own benefit
+1. Pedophiles using the AI Horde anonymously to generate CSAM images.
+1. Actors hating the ability for everyone to access generative AI for free,
+   and using attacks like DDOS or other vectors, to take down the service.
+
+### Countermeasures
+
+1. [The Kudos system](https://dbzer0.com/blog/the-kudos-based-economy-for-the-koboldai-horde/)
+   is built to prioritize people who are contributing back
+   to the horde. People trying to generate without contributing anything back,
+   will quickly find themselves at the back of the queue. As such, for-profit
+   leechers will quickly frustrate their users. On top of that, when we discover
+   any such services, we quickly steal their users by informing them they can
+   do the same on our free frontends, and faster.
+1. This is an area that I’ve put work-weeks of work into, to prevent and
+   discourage. I have even developed a ground-breaking technology of my own which
+   is being picked up to evaluate by researchers right now. You can read about
+   it [on my devlog](https://dbzer0.com/blog/ai-powered-anti-csam-filter-for-stable-diffusion/).
+1. This is an area that we are still potentially very vulnerable to. I have no
```

Comment:
We will need to spend time understanding the risk here, especially as Nivenly begins to help manage various infrastructure.